helm_deploy/values-dev.yaml

```diff
@@ -29,7 +29,7 @@ generic-service:
     CACHES_STAFFMEMBERS_EXPIRY-SECONDS: 20
     DOMAIN-EVENTS_CAS1_EMIT-ENABLED: true
     DOMAIN-EVENTS_CAS2_EMIT-ENABLED: true
-    DOMAIN-EVENTS_CAS3_EMIT-ENABLED: bookingCancelled,bookingConfirmed,bookingProvisionallyMade,personArrived,personDeparted,referralSubmitted,personDepartureUpdated
+    DOMAIN-EVENTS_CAS3_EMIT-ENABLED: bookingCancelled,bookingConfirmed,bookingProvisionallyMade,personArrived,personDeparted,referralSubmitted,personDepartureUpdated,bookingCancelledUpdated
```

Comment:
[We can now enable domain events on test and preprod straight away](https://mojdt.slack.com/archives/C03KVQUFSER/p1700566664986679?thread_ts=1699891958.767439&cid=C03KVQUFSER). Would you mind weaving that new requirement in before we merge? If not we can follow up with a small pr afterwards, either works.

(Changing the helm charts is perhaps not related to the OpenAPI spec as per this commit)

Reply:
Enabled them in test,preprod